src/AggregateFunctions/AggregateFunctionSparkbar.cpp

```diff
@@ -54,10 +54,16 @@ AggregateFunctionPtr createAggregateFunctionSparkbar(const std::string & name, c
     if (params.size() == 3)
     {
         if (params.at(1).getType() != arguments[0]->getDefault().getType() || params.at(2).getType() != arguments[0]->getDefault().getType())
-        {
             throw Exception("The second and third parameters are not the same type as the first arguments for aggregate function " + name, ErrorCodes::ILLEGAL_TYPE_OF_ARGUMENT);
-        }
     }
+
+    /// This value represents the width of the resulting sparkbar.
+    /// Later we will directly allocate width * sizeof(Float64) amount of memory,
```

Comment:
"which must be below 8GiB" ... I guess that's a limitation of msan and not of ClickHouse memory management in general. Suggest to mention that.